### PR TITLE
Church DLC: Pyre Edition

### PR DESCRIPTION
**Pull request by SELFHELL**
## About The Pull Request

Adds Pyres, a fun way of burning witches.

pyre shit
proper one

## Why It's Good For The Game

Good question. I guess campfires dont cut it anymore?

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/game/objects/structures/fluff.dm [hunk 1]
```diff
@@ -1234,6 +1234,45 @@
...
+	break_sound = 'sound/combat/hits/onwood/destroyfurniture.ogg'
+	attacked_sound = list('sound/combat/hits/onwood/woodimpact (1).ogg','sound/combat/hits/onwood/woodimpact (2).ogg')
+	density = FALSE
+	anchored = TRUE
+	blade_dulling = DULLING_BASHCHOP
+	layer = BELOW_MOB_LAYER
+	max_integrity = 100
+	sellprice = 40
+	flags_1 = HEAR_1
+	var/chance2hear = 30
```
Comment (ThePainkiller) [inline code/game/objects/structures/fluff.dm:1250]: Is this a duplicate copypaste variable, or is it floating and unused? I don't see it in use for...anything specific to pyres.

Comment (SELFHELL) [reply code/game/objects/structures/fluff.dm:1250]: good catch on that

**Comment by ThePainkiller** [comment]
These would be a whole lot cooler if you HAD to light them rather than being perpetually running bonfires. It's possible, might just need extra sprites, and a refactor into a light structure children (bonfire, oven etc) that takes fuel to run out of

**Comment by SELFHELL** [comment]
> These would be a whole lot cooler if you HAD to light them rather than being perpetually running bonfires. It's possible, might just need extra sprites, and a refactor into a light structure children (bonfire, oven etc) that takes fuel to run out of

Done

**Review by ThePainkiller — APPROVED** [review]
This will be great for public executions and an inquisitor RP buff

**Pull request merged**